### DISCUSSION TITLE
How to get more info about the project & how to contribute it?

### DISCUSSION THREAD

**Discussion by darkduckgo** [alibaba/GraphScope#897] [Q&A]
Due to the link of [Slack channel ](http://slack.graphscope.io/) is outdated, cannot find the proper way to communicate with community (So ask here first):
1. how to get the roadmap & todo list? (so people could know what issue/part can be addressed by themselves) like `help wanted` label:
   - feature (scheduled but not in progress)
   - docs (improve / update it)
   - bugfix (find but not assign to fixed now)
   - others
2. do we have a official blog / account to introduce the arch/feature/source code/paper/user case...etc, some main problem likes:
   - "Is gs(graphscope )under GA/stable state?"
   - "How much graph elements(vertice & edge) data does GS support ?"
   - "The relationship between pegasas and GAIA(x), and the relationship between GAIA(x) and GraphScope execution engine(GIE?)"

PS: Due to DD cannot support login on website by password, hope have better way to communicate.

**Comment by darkduckgo** [comment]
anyone could give some advice maybe? @BingqingLyu  @sighingnow  @yecol

**Comment by yecol** [comment]
Hi @darkduckgo , thanks for your interest in our work! 

1. We have a pinned issue, briefly describe the plan for the upcoming release. For long-term roadmap, we will open issues for discussion and add a section somewhere (e.g., readme).
2. GraphScope has a website at graphscope.io, as well as docs and blogs in the site. Hope them can help. For the unanswered and some specific questions like you listed here, you may ask in this `discussion` or reach us via DingTalk (group id = 31533139)

Thanks again for your suggestions to GraphScope community! We will add more friendly issues for new members of the community!

**Comment by imbajin** [comment]
@yecol nice to see the great work and glad to contribute something for GS:
- I wrote a **personal** zh-cn version of GAIA [here](http://imbajin.com/2021-10-14-Gremlin%E5%B9%B6%E8%A1%8C%E8%AE%A1%E7%AE%97%E7%B3%BB%E7%BB%9FGAIA%E8%AE%BA%E6%96%87%E4%BA%8C/), however not sure about its correctness, it would be nice if the community could give some advice or review it. (also could contribute to the community if needs)
- wonder to know whether `GAIA/GAIAX` can replace the original `Tinkerpop-Gremlin` in existing GraphDB(like gaia on JanusGraph etc.), and if possible:
  1. how much work will there be?
  2. shall I study the code for gaiaX directly? (or choose gaix) 
  3. does the community plan to extract the interface for direct use of the graph database?
     - if not, will the community be willing to help adapt together?

Hope to learn more and give feedback to the community in the future~ 🍇
